apps/web/src/pages/Home/IAPHomePage/Home.tsx

```diff
@@ -490,17 +535,28 @@ export const Home = ({ latestPool }: HomeProps) => {
           backgroundSize: "cover",
           backgroundPosition: "top center",
         }}
+        role="img"
+        aria-label={intl.formatMessage({
+          defaultMessage: "Traditional Métis sash",
+          id: "g6Atld",
+          description: "Indigenous Apprenticeship sash image text alternative",
+        })}
       >
         <div data-h2-container="base(center, iap-home, x1) p-tablet(center, iap-home, x2)">
           <div>
             <div data-h2-shadow="base(l)" data-h2-flex-grid="base(stretch, 0)">
               <div data-h2-flex-item="base(1of1) p-tablet(1of3) l-tablet(1of2) desktop(4of7)">
-                <div
+                <img
+                  src={lowerBack}
+                  alt={intl.formatMessage({
+                    defaultMessage: "Male Traditional dancer in regalia.",
+                    id: "9VPBwR",
+                    description:
+                      "Indigenous Apprenticeship lower back image text alternative",
+                  })}
                   data-h2-height="base(40vh) p-tablet(100%)"
                   style={{
-                    backgroundImage: `url(${lowerBack})`,
-                    backgroundSize: "cover",
-                    backgroundPosition: "center",
+                    objectFit: "cover",
```

Comment:
Hmm, why did this one get taller?

Before:
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/b2e7463e-a331-4325-9f56-c716a7f03ce4)

After:
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/0ca41a8f-d79d-432c-82ed-7ab06f8944d4)

Reply:
Object fit with 100% width results in displaying the whole image height while fitting the width. I'm not sure of an easy way around this without doing some tricky `position: absolute;` stuff 😬

Reply:
Brooke confirmed that the left side should zoom the image to match the height of the right side.

Reply:
Nice, the grid is much closer to the previous layout in mobile view now.
[switch dancer to grid](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8033/commits/bc38c5a4fcaebe6a0500261e6a6d76e64f7d2f08)

Reply:
Super nit picky, but I see a slight gap difference in mobile view. I tried a few things but couldn't get it exactly like it was previously. However, we could just expand to full width which is just an easy change to line 532: 

```js
<div data-h2-container="base(center, iap-home, **x2**) p-tablet(center, iap-home, x2)">
```

Before:
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/22059495/c3020bc4-6577-4726-a600-3871dcdca54c)

After:
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/22059495/e6e7c3d5-cdfa-4d19-a918-560df525e2fe)

Reply:
OK, I think I solved it using minmax.  It stays balanced down to about 140px - same as before.
[minmax for small grid](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8033/commits/28b82332183f591f595cc14ba805364cf83175c6)